llvm/lib/Target/AIE/AIE2GenInstrInfo.td

```diff
@@ -94,6 +94,8 @@ let Itinerary = II_NE in {
 }
 let Itinerary = II_OR in {
   def OR : AIE2_alu_r_rr_inst_alu<0b0101, (outs eR:$mRx), (ins eR:$mRx0, eR:$mRy), "or", "$mRx, $mRx0, $mRy">;
+  let isCodeGenOnly = 1 in 
```

Comment:
I think you should also set `let isMoveReg = 1;` to make possible to copy propagate across this instruction. You can also take a look in isCopyInstrImpl to see if it will be necessary to cover this case explicitly (or x, y, y).

Reply:
This specific instruction is not seen until we are finally scheduling, and thus we don't need isMoveReg. However, we need to make sure the multi slot instruction that has this opcode/inst as its alternative has isMoveReg=1 which it does for copy propagation across institutions to be possible.